plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterRating.php

```diff
@@ -197,12 +193,8 @@ private function render_rating_label( $rating, $count_label ) {
 		?>
 		<div class="wc-block-components-product-rating">
 			<div class="wc-block-components-product-rating__stars" role="img" aria-label="<?php echo esc_attr( $rating_label ); ?>">
-				<span style="width: <?php echo esc_attr( $width ); ?>%" aria-hidden="true">
-				</span>
+				<span style="width: <?php echo esc_attr( $width ); ?>%" aria-hidden="true"></span>
```

Comment:
Should this specify units? ie px?

Reply:
The rating size can be vary, so we need to use relative unit here.